Userland/Libraries/LibTest/TestCase.h

```diff
@@ -42,34 +42,43 @@ void add_test_case_to_suite(NonnullRefPtr<TestCase> const& test_case);
-    struct __TESTCASE_TYPE(x) {                                                                         \
-        __TESTCASE_TYPE(x)                                                                              \
-        () { add_test_case_to_suite(adopt_ref(*new ::Test::TestCase(#x, __TESTCASE_FUNC(x), false))); } \
-    };                                                                                                  \
-    static struct __TESTCASE_TYPE(x) __TESTCASE_TYPE(x);                                                \
+#define TEST_CASE(x)                                                                                 \
+    static void __TESTCASE_FUNC(x)();                                                                \
+    struct __TESTCASE_TYPE(x) {                                                                      \
+        __TESTCASE_TYPE(x)                                                                           \
+        ()                                                                                           \
+        {                                                                                            \
+            add_test_case_to_suite(adopt_ref(*new ::Test::TestCase(#x, __TESTCASE_FUNC(x), false))); \
+        }                                                                                            \
+    };                                                                                               \
+    static struct __TESTCASE_TYPE(x) __TESTCASE_TYPE(x);                                             \
     static void __TESTCASE_FUNC(x)()
 
 #define __BENCHMARK_FUNC(x) __benchmark_##x
 #define __BENCHMARK_TYPE(x) __BenchmarkCase_##x
 
-#define BENCHMARK_CASE(x)                                                                               \
-    static void __BENCHMARK_FUNC(x)();                                                                  \
-    struct __BENCHMARK_TYPE(x) {                                                                        \
-        __BENCHMARK_TYPE(x)                                                                             \
-        () { add_test_case_to_suite(adopt_ref(*new ::Test::TestCase(#x, __BENCHMARK_FUNC(x), true))); } \
-    };                                                                                                  \
-    static struct __BENCHMARK_TYPE(x) __BENCHMARK_TYPE(x);                                              \
+#define BENCHMARK_CASE(x)                                                                            \
+    static void __BENCHMARK_FUNC(x)();                                                               \
+    struct __BENCHMARK_TYPE(x) {                                                                     \
+        __BENCHMARK_TYPE(x)                                                                          \
+        ()                                                                                           \
```

Comment:
clang-format really doesn't know what to do with these constructors, does it :D macro overload. no change requested.